README.md

```diff
@@ -1,8 +1,8 @@
-# Android Messaging SDK v5.3.0
+# Android Messaging SDK v5.4.0
 
-In-App Messaging SDK v5.3.0 for Android includes the same change as 4.5.0:
-* Hyperlink support in messaging
-* Security element encryption improvements.
+In-App Messaging SDK v5.4.0 for Android includes the same change as 4.6.1:
+* Auto logout
+* Changed retry mechanism. Notify host app immediately when there is an error.
```

Comment:
Can we say something like this:
"Notify host app immediately when failed to authenticate consumer instead of performing periodic reties"